### ISSUE TITLE
Proguard error in 1.0.12

### ISSUE THREAD

**Issue by okmanideep** [freshdesk/hotline-android#6]
## TLDR

add `-dontwarn uk.co.chrishenx.calligraphy.**` in your proguard rules

and change your README guide to use fixed version rather than `1.0.+`
## What happened

Pulled the `develop` branch **BUILD FAILED**

ARRGGHHH some proguard sh*t

Try adding some extra rules for updated libraries like auto value

Lot of Googling about auto value proguard rules

ARRGHH this proguard!!!

More googling. Nothing seems to work

Pulled the `master` (the version in the playstore) **BUILD FAILED**

WHAT!!! HOW!!! 

Checked warnings from dynamic dependencies and figured it out.
## Moral of the story

**Don't use dynamic dependencies**. Lesson learnt the hard way

**Comment by tryprasannan** [comment]
@manidesto Thanks for bringing this to our notice and sorry for the inconvenience.
We will consider ways to handle this better. Didn't want to force a dependency on everybody as not everybody needs/uses this.

**Comment by tryprasannan** [comment]
This is fixed in [1.0.13](https://github.com/freshdesk/hotline-android/releases/tag/1.0.13)
